spec/spec_helper.rb

```diff
@@ -1,2 +1,8 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'trix'
+require 'rails/version'
+require 'action_dispatch'
+require 'action_controller/metal/live'
+require 'action_controller/test_case'
```

Comment:
Actually `rails/version` is no longer needed, I was initially requiring some things conditionally but no longer need to by dropping rails 3 support). As for the rest, `rspec/rails` needs them, it could be simplified to `require 'action_controller'` though.